Directory.Packages.props

```diff
@@ -25,6 +25,9 @@
     <PackageVersion Include="Rampastring.XNAUI.UniversalGL.Debug" Version="$(RampastringXNAUIVersion)" />
     <PackageVersion Include="Rampastring.XNAUI.WindowsXNA.Debug" Version="$(RampastringXNAUIVersion)" />
     <PackageVersion Include="SixLabors.ImageSharp" Version="2.1.9" />
+    <PackageVersion Include="SteamworksSharp" Version="1.0.8" />
+    <PackageVersion Include="SteamworksSharp.Native" Version="1.0.8" />
+    <PackageVersion Include="SteamworksSharp.Native.Windows_x64" Version="1.0.8" />
```

Comment:
![image](https://github.com/user-attachments/assets/c600d6e1-7750-478e-a247-f34a5986f371)
![image](https://github.com/user-attachments/assets/40723010-870b-44bb-b770-0dcbf8744601)

Uhh I don't think we should be using this. This is a pretty random wrapper wrote by a random person 6 years ago.

For a 1:1 Steamworks mappings use `Steamworks.NET`. For a preoprly C# one I think `Facepunch.Steamworks` should be used (based on some quick reading up I would pick that, even if it's outdated). Please research this question a bit more.